Islandora-Devops/claw-playbook
# update to 8.3, solr 7, make it stand alone

**GitHub Issue**: (link)
https://github.com/Islandora-CLAW/CLAW/issues/1163
https://github.com/Islandora-CLAW/CLAW/issues/1197

# What does this Pull Request do?
* Installs and configures search_api_solr 8.3
* Supports installing solr on its own server.


# How should this be tested?
* Get the PR
* vagrant up
* Create content
* Verify that it gets indexed and returned 

In addition, testing the following is helpful.
* Get the PR
* Modify or create the following playbook:
```
---

- include: bootstrap.yml
  tags:
    - bootstrap
- include: solr.yml
```
* deploy and check that solr has been correctly installed

# Additional Notes:
Although the above works, " You are using outdated Solr configuration set. Please follow the instructions described in the INSTALL.md file for setting up Solr. " is being reported here: http://localhost:8000/admin/config/search/search-api/server/default_solr_server

Not sure what may be cause of error.  

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora-CLAW/committers
